WordPress/Classes/Utility/DisplayableImageHelper.m

```diff
@@ -84,9 +85,15 @@ + (NSString *)searchPostContentForImageToDisplay:(NSString *)content
         if (width > currentMaxWidth) {
             imageSrc = src;
             currentMaxWidth = width;
+        } else if (!firstImageWithNoSize && width == 0) {
+            firstImageWithNoSize = src;
         }
     }
 
+    if ([imageSrc length] == 0 && firstImageWithNoSize) {
```

Comment:
Just a note, and kind of unrelated to this PR but I'm mentioning this anyway because it's adjacent to your code.

The original code in this method uses `@""` (empty string) to mean no-string.  While this may work here, it's a bad idea to equate an empty string with the absence of a chosen string since it may not be true in all methods - hence not being a good standard.

Just thought it was worth mentioning this.  It doesn't need to be changed in this PR.